packages/kbn-es/src/custom_snapshots.js

```diff
@@ -27,7 +27,8 @@ function getCustomSnapshotUrl() {
   // force use of manually created snapshots until live ones are available
   if (!process.env.KBN_ES_SNAPSHOT_URL && !process.argv.some(isVersionFlag)) {
     // return 'https://storage.googleapis.com/kibana-ci-tmp-artifacts/{name}-{version}-{os}-x86_64.{ext}';
-    return undefined;
+    // return undefined;
```

Comment:
these do expire, but i think we can assume we'll within the time frame

Reply:
I'm moving it to non expiring artifacts on the GCS bucket